replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+        axis.title.y = element_text(face = 'bold', size = 15), 
+        plot.title = element_text(face = 'bold', size = 20),
+        legend.title = element_text(face = 'bold', size = 12),
+        legend.text = element_text(size = 12),
+        axis.text.x = element_text(size = 12), 
+        axis.text.y = element_text(size = 12))
+```
+
+(4.1.1) Code for graph and moderation analyses:
+
+```{r eval = FALSE}
+
+# Graph the number of participants who chose each pen, by music condition:
+ggplot(dat_egood, 
+       aes(x = music, fill = chose_advertised_pen)) +
+  geom_bar(stat = "bin",
+           position = position_dodge()) + 
+  scale_fill_brewer(name = "Pen Choice", 
+                    labels = c("Not Advertised", "Advertised"), palette = "Set2") +
+  ggtitle("Frequency of Pen Choice By Condition") +
+  ylab("Frequency") +
+  xlab("Music Conditon") +
+  scale_x_discrete(breaks=c("1", "2"), 
+                   labels=c("Liked", "Disliked")) +
+  theme_bw() + 
+  theme(axis.title.x = element_text(face = 'bold', size = 15),
+        axis.title.y = element_text(face = 'bold', size = 15), 
+        plot.title = element_text(face = 'bold', size = 20),
+        legend.title = element_text(face = 'bold', size = 12),
+        legend.text = element_text(size = 12),
+        axis.text.x = element_text(size = 12), 
+        axis.text.y = element_text(size = 12))
+
+```
+
+(4.2) Extension B: Investigating the possible moderation of effects by gender, using a logistical regression.
+
+Since the musical stimulus chosen for manipulation in this study was performed by a female artist, it is possible that the strength of aversion to the advertised pen (when paired with a disliked music) would be different by gender. Note: We are quite underpowered to conduct this test as a confirmatory analysis, and there is little theoretical precedent for looking at gender differences in pen choice; thus, this was an entirely exploratory investigation. It is possible that the disliked (and distorted sound) of the female singer's voice impacted women more so than men, such that women had an even greater liklihood of choosing the non-advertised pen when assigned to the disliked-music conditioin. However, as displayed in our analyses of the predicted probabilities below, first for the entire sample, and then for the data subsetted by gender, the likelihood of men and women chosing the advertised or non-advertised pen in each condition is nearly identitical. A greater sampling of men (as well as women) would be needed in order to test this hypothesis with greater statistical power.
+
+(4.2.1) A quick check of N by gender in data:
+
+```{r echo = FALSE}
+
+#Get counts and check values against article for correct coding scheme:
+count(dat_egood, geslacht1)
+#confirmed:
+## 1 = male
+## 2 = female
+
+```
+
+(4.2.2) Code for logistical regressions:
+
+```{r eval = FALSE}
+
+#Recode dummy variable for music as 0 and 1 for easier interpretation of estimates:
```

Comment:
this is good, try to use it earlier in the code too (where gender was coded 1 and 2)